utils/common/src/main/kotlin/Extensions.kt

```diff
@@ -236,8 +237,17 @@ fun JsonNode?.textValueOrEmpty(): String = this?.textValue().orEmpty()
  * Merge two maps by iterating over the combined key set of both maps and calling [operation] with any conflicting
  * values for the same key.
  */
-inline fun <K, V> Map<K, V>.zip(other: Map<K, V>, operation: (V, V) -> V): Map<K, V> =
-    (keys + other.keys).associateWith { key ->
+inline fun <K, V> Map<K, V>.zip(other: Map<K, V>, operation: (V, V) -> V): Map<K, V> {
```

Comment:
Please add to the docs that the order of sorted maps is kept.

Reply:
By "order of sorted maps" you mean the iteration order over the keys, right?